Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+RUN apt-get update -y && \
+    pip install --upgrade pip setuptools
```

Comment:
As `python` image already has `pip` I think we may skip step to update deb packages, pip and setuptools.